### ISSUE TITLE
Barrier method produces spurious messages as side effects

### ISSUE THREAD

**Issue by ralacroix** [coin-or/Clp#184]
When using CLP Barrier method via OSI, there's no way to suppress all the messages, so CLP messages appear in the application's output. Despite using OsiClpSolverInterface->setLogLevel(0) each call to the solver produces messages like

0 primal infeasibilities summing to 0
29595 dual infeasibilities summing to 6.25167e+09

The messages do not appear when using the Simplex method.

This is in CLP 1.17.3

**Comment by jjhforrest** [comment]
No idea how you managed to get those messages.

They seem to be in OsiClpSolverInterface::crossover which is not called 
from anywhere and was probably experimental.

Have fixed in master.

**Comment by ralacroix** [comment]
I see what happened. It's our code which called crossover(), and it was on the wrong side of "#ifdef experimental" in a section using the Barrier method. That explains everything. I've stopped it from calling crossover(), and that cleans up the messages on our end. It won't impact our results because we haven't been using Barrier method (until now). But maybe we will now that the problem with spurious messages has been solved.
